app/controllers/admin/users_controller.rb

```diff
@@ -27,9 +27,20 @@ def destroy
     redirect_to admin_users_path
   end
 
-  def blacklist
-    @user.update_attributes(blacklisted: true)
-    flash[:notice] = "User was blacklisted"
+  def block
+    @user.update_attributes(is_blocked: true)
+    @coach = Coach.where(user_id: @user.id).first
+    @coach_applications = CoachApplication.where(coach_id: @coach.id)
```

Comment:
Should we make the changes to the coach applications only for future events? Otherwise we will be changing history.